File: backend/src/main/java/com/gymapp/modules/membership/dto/UpdateUserGoalRequest.java
package com.gymapp.modules.membership.dto;

import com.gymapp.modules.membership.enums.ExperienceLevel;
import com.gymapp.modules.membership.enums.FitnessGoal;

import lombok.Data;
import lombok.Getter;
import lombok.Setter;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.NoArgsConstructor;

@Data
@Getter
@Setter
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class UpdateUserGoalRequest {
    private ExperienceLevel experienceLevel;
    private FitnessGoal fitnessGoal;
}

File: android/app/src/main/java/com/gymapp/android/data/remote/dto/user/UpdateUserGoalRequest.kt
package com.gymapp.android.data.remote.dto.user

import com.gymapp.android.domain.model.goal.ExperienceLevel
import com.gymapp.android.domain.model.goal.FitnessGoal

data class UpdateUserGoalRequest(
    val experienceLevel: ExperienceLevel,
    val fitnessGoal: FitnessGoal
)